ajax/libs/stampit/package.json

```diff
@@ -1,39 +1,37 @@
 {
   "name": "stampit",
-  "version": "2.1.0",
+  "version": "3.0.0",
   "description": "Create objects from reusable, composable behaviors.",
   "author": {
     "name": "Eric Elliott",
     "url": "https://ericelliottjs.com"
   },
+  "license": "MIT",  
   "keywords": [
     "object",
     "prototype",
     "object oriented",
     "browser",
     "inheritance",
-    "oo",
+    "oop",
     "node",
     "factory",
-    "class"
+    "class",
+    "stamp"
```

Comment:
Well these are keyword from official repo, so perhaps first add it there and then we can go with it here :)